apps/lib/blockly/tg_tj.js

```diff
@@ -1,14 +1,14 @@
 Blockly.Msg.ACTUAL = "call";
-Blockly.Msg.ADD = "Илова";
-Blockly.Msg.ADD_COMMENT = "Иловаи шарҳ";
-Blockly.Msg.ADD_PARAMETER = "Иловаи параметр";
+Blockly.Msg.ADD = "Add";
+Blockly.Msg.ADD_COMMENT = "Add Comment";
```

Comment:
This file reverting to English is wrong.

https://crowdin.com/translate/codeorg/54/enus-tg#3187